### DISCUSSION TITLE
Score import from outdated beatmap fails

### DISCUSSION THREAD

**Discussion by nzxl101** [ppy/osu#16819] [Q&A] [answered]
I don't know if this is a duplicate, but when importing an outdated beatmap and trying to retrieve the score replay it just fails with the error `⚠️ No corresponding beatmap for the score could be found.`.
It doesn't seem like the game recognizes that the map is outdated because ranked status and other stuff is showing correctly due to the web api, I guess?

https://user-images.githubusercontent.com/63413558/152897141-59818343-e899-4041-b252-4eb1af31a101.mp4

**Comment by frenzibyte** [comment, marked as answer]
You're seeing that error message because the beatmap hash embedded within the downloaded score no longer matches the one you currently have (due to yours being outdated).

Identifying outdated maps and updating is tracked under https://github.com/ppy/osu/issues/4823.
